":read :cmd" reads Ex command ":cmd" output (#30628)
neovim/neovim: runtime/doc/news.txt

  "~/" are now expanded to the user's profile directory, not a relative path
  to a literal "~" directory.
• |hl-PmenuMatch| and |hl-PmenuMatchSel| show matched text in completion popup.

> • |:read:| can now execute an Ex command and insert the output into the buffer.

Reviewer: ```suggestion
• |:read:| reads the output of an Ex command and into the buffer. Example:>
  :read :ls
```

  "~/" are now expanded to the user's profile directory, not a relative path
  to a literal "~" directory.
• |hl-PmenuMatch| and |hl-PmenuMatchSel| show matched text in completion popup.

> • |:read:| reads the output of an Ex command into the buffer. Example:> :read
>   :ls

Reviewer: ```suggestion
• |:read:| reads the output of an Ex command into the buffer. Example:>
    :read
    :ls
<
```
